test/server/graphql/v1/expenses.test.js

```diff
@@ -1272,7 +1272,7 @@ describe('server/graphql/v1/expenses', () => {
           hostFeeInCollectiveCurrency +
           platformFeeInCollectiveCurrency;
         expect(balance).to.equal(initialBalance - expensePlusFees);
-        await utils.waitForCondition(() => emailSendMessageSpy.callCount === 4);
+        await utils.waitForCondition(() => emailSendMessageSpy.callCount === 2);
```

Comment:
Because the `createActivity` was previously in the `afterUpdate` hook, we were sending two more emails here for "expense approved". This was not correct because the expense was already approved before the update (see https://github.com/opencollective/opencollective-api/pull/3653/files#diff-35fd0807cc45439c1187333c6f4c30d6R1239) so there's no reason to send these two emails.